exercises/collatz-conjecture/canonical-data.json

```
@@ -54,6 +65,26 @@
         "number": -15
       },
       "expected": { "error": "Only positive numbers are allowed" }
+    },
+    {
+      "uuid": "ec11f479-56bc-47fd-a434-bcd7a31a7a2e",
+      "description": "negative value is an error",
+      "comments": ["Collatz Conjecture holds only for positive integers"],
+      "reimplements": "c6c795bf-a288-45e9-86a1-841359ad426d",
+      "property": "steps",
+      "input": {
+        "number": -15
+      },
+      "expected": { "error": "Only positive integers are allowed" }
+    },
+    {
+      "uuid": "f7176864-e35d-11eb-ba80-0242ac130004",
```

This change should fix the second CI failure:
```suggestion
      "uuid": "f7176864-e35d-41eb-ba80-0242ac130004",
```